### ISSUE TITLE
State "isCurrentPage" & "isInRootline" missing in 3rd depth level and below

### ISSUE THREAD

**Issue by rintisch** [b13/menus#31]
Having a menu with depth=7 I have the problem that `isCurrentPage` and `isInRootline` are not given for pages with depth level 3 and below (so 4th, 5th, ...).

Here is a screenshot of the debug list. 
As you can see the first and second level do have both attributes but the third does not:

![grafik](https://user-images.githubusercontent.com/44193208/93421244-1a28c200-f8b1-11ea-97d9-eff427474669.png)

The TypoScript setup is the following:

     50 = B13\Menus\DataProcessing\TreeMenu
      50 {
        entryPoints = {$plugin.tx_extensionname.settings.mainMenuPid}
        depth = 6
        excludePages = {$plugin.tx_extensionname.settings.pageToExcludePid}
        as = mainnavigation
      }

Is this a bug or is there something that I could have made wrong?

**System:**
TYPO3 9.5.21
EXT:b13/menus v0.2.0

**Comment by rintisch** [comment]
I found the bug but also saw that is was already fixed 8 months ago with this commit: https://github.com/b13/menus/commit/f7435cfb64b591f90a505796486fc8e44f4aee88

I don't understand why I am got an old version. This happened the second time with a repository from B13 loaded with composer (first time was with `container` extension). The version did not change. Might this be a reason?

Snapshot from `composer.json`
![grafik](https://user-images.githubusercontent.com/44193208/94104993-9e8ebe00-fe38-11ea-99ee-30a6f86c8b4d.png)

Snapshot from `PageStateMarker.php` -> This is not the latest version, I just made a `composer update` but it's not fetching the real latest version. 
![grafik](https://user-images.githubusercontent.com/44193208/94105091-d4cc3d80-fe38-11ea-9471-a71cb458821c.png)


What am I doing wrong?
